.nxxm/deps

```diff
@@ -1,10 +1,17 @@
 {
-   "daminetreg/js-bind" : { "@" : "v0.0.2" }
+   "nxxm/js-bind" : { "@" : "v0.0.0" }
```

Comment:
oh!!! v0.0.3

Reply:
On nxxm/js-bind there was no version so I created the first version of se repo so v0.0.0. So do you want me to upgrade it to v0.0.3 ?

Reply:
I'm doing it right now, there were 3 version : 

<img width="1279" alt="image" src="https://user-images.githubusercontent.com/181726/106801601-3f507e00-6662-11eb-9c03-53613e17039a.png">